### ISSUE TITLE
Type alias declaration error with dictionaries

### ISSUE THREAD

**Issue by rmonat** [nohtyprm/MrPython#106]
This code is rejected: 

```python
# type Base = dict[str: float]
# b : Base
b = {'a':1.0}
```

With a really strange error:
```Erreur: ligne 1
==> Problème de déclaration: The declared type for variable 'typeBase=dict[str' has strange appended string: ]```

I think this is valid and should be accepted (useful for chapter 9 exercises...).

**Comment by fredokun** [comment]
I've just tested with master (3.0.6-beta) and it works... Maybe the space after `str:` is not recognized by the lexer?

**Comment by fredokun** [comment]
Ok I could reproduce the bug. The following *is* working :

```python
# type Base = dict[str: float]

# b : Base
b = {'a':1.0}
```

I think the parser expects two newlines instead of one after an alias declaration

**Comment by fredokun** [comment]
I have an explanation, and it's a little bit "ugly"... In fact, the problem is when parsing the variable declarations, the parser tries to parse `# type Base = ...` as a variable declaration... which it is not of course. I think the fix in 91cf7ad08c2386976ce61398490de3bc0abd3bd7 solves the issue, but it's a little bit "hackish".
